### ISSUE TITLE
MacOS Frequent Interface De-selection Bug

### ISSUE THREAD

**Issue by Promax8** [aseprite/aseprite#4681]
While drawing or doing anything in the main canvas, the tabs or anything else outside the main canvas get "unselected," and you need to click on them twice to select them. This is especially annoying if you have to switch between multiple tabs often or don't use shortcuts for everything (so you have to click a button), as in that case, it usually puts a pixel on the canvas, or if you have the paint bucket tool selected, it fills the whole canvas with color, so you have to undo. I reinstalled Aseprite three times, and the issue still remains, so it can't be some weird file on my Mac that I'm not seeing. It would be great if this is fixed soon; it feels horrible to use since upgrading to the latest version. Thanks!

Example Video:
https://cln.sh/RSt4VGZD

### Aseprite and System version

* Aseprite version: v1.3.8.1
* System: Macos Sonoma 14.7 (23H124)

**Comment by emilpixels** [comment]
it in fact did not fix it

**Comment by dacap** [comment]
Hi @emilpixels, are you using just a mouse or a stylus? Did older versions work correctly? Would you mind to contact us at support@aseprite.org?

**Comment by emilpixels** [comment]
Hey, contacted support 4 days ago, don't know how long it usually takes, but if it takes longer, I'll be patient. Just not sure if the message reached y'all.
I use a mouse btw and older versions worked properly. The last one I remember actively using is 1.3, but could be wrong there.

**Comment by dacap** [comment]
Hi @emilpixels, we've received your message. What specific mouse are you using? Are you using some special driver for your mouse?

**Comment by emilpixels** [comment]
Hey, I'm using the Logitech G Pro X Superlight (Version 1) and have been using it for like 2 years and that bug never occurred before. I'm using Logitch G Hub (don't know if that automatically installs the drivers, but I didn't install them manually)
Hope this helps!
(Also, is it possible for me to download an older version of asprite so I can have a better workflow until this is fixed?

**Comment by Gasparoken** [comment]
Hi @emilpixels. If the 'Flow' feature in 'Logi Options' is enabled, could you try disabling it?
![Oct-01-2024 17-00-48](https://github.com/user-attachments/assets/7eea2d23-4db3-4128-8bee-75ea25b1e65d)

**Comment by emilpixels** [comment]
I don't seem to have that option with my mouse
![JOnRbOhr-01-10-2024@2x](https://github.com/user-attachments/assets/6f797be0-17a4-4457-915b-66dd5a24a807)

**Comment by emilpixels** [comment]
I just tried using my trackpad and another mouse and the same error happens when I'm using those, so it probably can't be hardware related

**Comment by emilpixels** [comment]
any news on this, I rlly need it fixed pls 😭

**Comment by dacap** [comment]
I'll try to send you a previous version today so you can check if it's a new issue in Aseprite or in macOS.

**Comment by emilpixels** [comment]
@dacap I've tampered a little with the different versions you sent me and also deleted the Aseprite folder in application support multiple times to see if that changes something. Sadly, it did not. I found out though, that below a certain window size of the application, the bug stops accuring and everything works fine. I'll attach a video. Hope this helps!

https://cln.sh/HdcKlj2W

(As you can see in the video, the cursor stays in the "drawing" mode when clicking when the window size gets larger than a certain point)

**Comment by emilpixels** [comment]
Hey, did anyone have a look at this yet :)

**Comment by dacap** [comment]
@emilpixels just answer your mail, we've sent you a couple of older versions to your email address, please give a try to check if this problem is an older problem (if this problem persists in older versions, it means that might be a new issue on macOS 15.0 and not a change that we did)

**Comment by boubl** [comment]
I have the same problem on macOS 15.3.2 with Aseprite 1.3.13, just upgraded from 1.3.7 to try to fix the bug but it's still here. It's been a while since I launched Aseprite so I don't know which macos update made this bug.

**Comment by boubl** [comment]
hello @dacap i'm still having the bug and it's very very frustrating and makes Aseprite a real pain to use. I'll gladly help you find what's wrong with my installation if you want because I would really like to use this software normally again 🥲

**Comment by martincapello** [comment]
@boubl would you mind giving us some information about your display configuration? Resolution, refresh rate, if it is the built-in display or an external one...

**Comment by emilpixels** [comment]
I'm still having the bug btw, I'm just clicking outside of aseprite and then clicking what I wanna click. its shit but I got used to it

**Comment by martincapello** [comment]
> I'm still having the bug btw, I'm just clicking outside of aseprite and then clicking what I wanna click. its shit but I got used to it

Are you using Aseprite in full screen or it does happen regardless of using the full screen mode? Also, which Aseprite version are you using right now?

**Comment by martincapello** [comment]
There is a chance that the next release includes a [fix](https://github.com/aseprite/laf/pull/131) that hopefully will solve this issue. At least I was able to reproduce this issue only using Aseprite full screen mode, but when using the fixed version, it works as expected.

**Comment by martincapello** [comment]
> I do not need to enable fullscreen for it to happen.

This is getting more "interesting"...I have just tried using different resolutions on my display, and when tried 1920x1200, this issue happens without the need of going full screen. Then I tried the fixed version, and it worked as well, so now I'm pretty confident that the fix we have should fix this issue.

**Comment by martincapello** [comment]
@emilpixels hope the fix I mentioned solves your issue as well.

**Comment by emilpixels** [comment]
I don't seem to understand the fix here, just wait for the new update?

**Comment by martincapello** [comment]
> I don't seem to understand the fix here, just wait for the new update?

Yes, it should be fixed in the next update, which should be soon.

**Comment by boubl** [comment]
I found the app messing up with Aseprite, it was NotchNook (widget-like app hidding in the notch). Killing the app fixes the issue.

**Comment by martincapello** [comment]
Wow, I 'm glad you figured out that! But I'm still wondering why I was able to reproduce it...maybe it was a different issue. Could you try full screen and tell me if the issue comes back?

**Comment by AaronC81** [comment]
Thanks for the pointers, everyone!

I was encountering the same issue, and it turned out to be caused by MediaMate, a similar "things in the notch" app.
After quitting MediaMate, my Aseprite is no longer losing focus all the time.
